authCommon/app/modules/dao/AllowedIssuerDao.js

```diff
@@ -163,9 +164,25 @@ async function listJwksCacheExpiringAtMinute(expiringMinute){
     return result.Items
 }
 
+async function postponeJwksCacheEntryValidation(hashKey, jwksCacheExpireSlot){
+    const updateItemInput = {
+        TableName: process.env.AUTH_JWT_ISSUER_TABLE,
+        Item: {
+            "hashKey": hashKey,
```

Comment:
@ViVeniero la chiave è composta da hashKey e sortKey, devi passarle entrambe altrimenti non funziona.

La sortKey in questo caso è "CFG"

Reply:
done